overwolf/src/Frame.tsx

```diff
@@ -76,7 +76,7 @@ export default function Frame(props: IProps) {
                     setAppContextSettings(prev => prev.iconSettings?.categories[categoryType[0]]?.types[categoryType[1]] !== undefined
                         ? produce(prev, draft => {
                             if (draft.iconSettings) {
-                                draft.iconSettings.categories[categoryType[0]].types[categoryType[1]].value = loadIconType(categoryType[0], categoryType[1]);
+                                draft.iconSettings.categories[categoryType[0]].types[categoryType[1]].visible = loadIconConfiguration(categoryType[0], categoryType[1], 'visible');
                             }
                         })
                         : prev);
```

Comment:
Opening a new thread to discuss the UI change

Here's some screenshots with the Red/Green/Grey system.

![unknown](https://user-images.githubusercontent.com/4268247/137662197-83b392c9-60a8-40d8-9eaa-ba93d307caac.png)

Please, share your thoughts. Honestly I'm not really convinced with the red/green, but I really love the contrast between the "disabled" label icon and the enabled/disabled visibility.

After we define the UI, I can go ahead an update the selector by category and the mass selection by category
![image](https://user-images.githubusercontent.com/4268247/137662327-9e68437e-e89a-4dbd-85e0-c2290a1f322b.png)

Reply:
I'm not sure about why there's a select/unselect all and a category checkbox, is not the same?

What happen is you can't enable/disable the category, but instead of that you have just the select/unselect all. Then, if you want to "hide" an category, you may unselect all, but if you want to see just one type, you can enable just this type. Im not sure if I'm being clear 🤔